src/node/datomish/cljify.cljs

```diff
@@ -1,7 +1,27 @@
 (ns datomish.cljify)
 
 (defn cljify
-  "In node, equivalent to `(js->clj o :keywordize-keys true).
-  See <http://dev.clojure.org/jira/browse/CLJS-439?focusedCommentId=43909>."
-  [o]
-  (js->clj o :keywordize-keys true))
+  "In node, equivalent to `(js->clj o :keywordize-keys true),
+  but successfully passes Clojure Records through. This allows JS API
+  callers to round-trip values they receive from ClojureScript APIs."
+  [x]
+  (if (record? x)
```

Comment:
It really looks like this is identical to https://crossclj.info/fun/cljs.core.cljs/js-%3Eclj.html, save for the `record?` check.  Can you comment on exactly the difference, and explain that the repetition is needed to get things to work recursively?